apps/http-status-codes/src/files/codes.js

```diff
@@ -0,0 +1,377 @@
+//data imported from MDN - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+const status_codes=[
+    {
+        value:"100",
+        meaning:"Continue",
+        description:"This interim response indicates that the client should continue the request or ignore the response if the request is already finished."
+    },
```

Comment:
Did you write all these yourself?

Reply:
Ya, I have manually copied and pasted the information from the MDN website😅

Reply:
Hard work! Thanks for putting in the time.